package.json

```diff
@@ -7,7 +7,9 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run"
```

Comment:
Should we add a small line in the `README.md` file to provide a quick introduction to these commands? (if yes, we can do the same for the Unit Tests command ;-) )

Reply:
sure
the above two commands are included in the README here: https://github.com/tyrro/react-survey/pull/47/commits/4e0fa630b33db2a061f0f2458d4a80d8866693e7 👍